src/components/header/BeautyDropdown.jsx

```diff
@@ -0,0 +1,330 @@
+import * as React from 'react';
+import './appbar.css';
+import Box from '@mui/material/Box';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import IconButton from '@mui/material/IconButton';
+import Tooltip from '@mui/material/Tooltip';
+import { styled } from '@mui/material/styles';
+import { makeStyles } from '@material-ui/core';
+import Grid from '@mui/material/Grid';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(() => ({
+  catContainerBeauty: {
+    paddingTop: '1.65em',
+    color: '#282c3f',
+    fontSize: '14px',
+    letterSpacing: '0.5px',
+    fontFamily:
+      'Whitney, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica, Arial, sans-serif',
+    fontWeight: '500',
+    textTransform: 'uppercase',
+    '&:hover ': {
+      borderBottom: '4px solid #0db7af',
+    },
+  },
+}));
+
+const MenuBold = styled(MenuItem)(() => ({
+  fontSize: '14px',
+  paddingTop: '2px',
+  paddingBottom: '2px',
+  paddingRight: '6px',
+  fontFamily:
+    ' Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif',
+  color: '#3e4152',
+  '&:hover': {
+    color: '#3e4152',
+    fontWeight: '700',
+    background: 'white',
+  },
+}));
+
+const MenuBold2 = styled(MenuItem)(() => ({
```

Comment:
Styling objects can be initialized to some other class and can be imported here as exported key name!!